docs/Code_Dependencies.md

```diff
@@ -0,0 +1,292 @@
+│   │       CompanyOptions.tsx
+│   │       DangerConfirmationModal.tsx
+│   │       LoadingMessages.tsx
+│   │       Pagination.tsx
+│   │       QuickAcessItems.tsx
+│   │       Search.tsx
+│   │       StepsBreadCrumbs.tsx
+│   │       TopMenu.tsx
+│   │       UnitsofMeasureOptions.tsx
+│   │
+│   ├───dcm
+│   │       AppComponent.tsx
+│   │       Layout.tsx
+│   │       ToastContainerComponent.tsx
+│   │
+│   ├───demands
+│   │       DemandAddForm.tsx
+│   │       DemandCategoryOptions.tsx
+│   │       DemandDetailsModal.tsx
+│   │       DemandEditForm.tsx
+│   │       DemandList.tsx
+│   │       DemandListTableHeaders.tsx
+│   │       DemandManagement.tsx
+│   │       DemandManagementTableHeaders.tsx
+│   │       DemandsOverview.tsx
+│   │
+│   ├───events
+│   │       EventsTable.tsx
+│   │
+│   ├───favorites
+│   │       FavoritesTableAddressBook.tsx
+│   │       FavoritesTableCapacityGroup.tsx
+│   │       FavoritesTableCompanies.tsx
+│   │       FavoritesTableEvents.tsx
+│   │       FavoritesTableMaterialDemands.tsx
+│   │
+│   ├───menu
+│   │       InfoMenu.tsx
+│   │
+│   └───pages
+│           AdminPage.tsx
+│           AdressBookPage.tsx
+│           AlertsPage.tsx
+│           AuthenticationPage.tsx
+│           CapacityGroupDetailsPage.tsx
+│           CapacityGroupPage.tsx
+│           DownStatusPage.tsx
+│           ErrorPage.tsx
+│           EventsPage.tsx
+│           FavoritesPage.tsx
+│           ThresholdPage.tsx
+│           TodoListPage.tsx
+│           UpStatusPage.tsx
+│
+├───contexts
+│       AdressBookContextProvider.tsx
+│       AlertsContextProvider.tsx
+│       BottlenecksContextProvider.tsx
+│       CapacityGroupsContextProvider.tsx
+│       CompanyContextProvider.tsx
+│       DemandCategoryProvider.tsx
+│       DemandContextProvider.tsx
+│       EventsContextProvider.tsx
+│       FavoritesContextProvider.tsx
+│       InfoMenuContextProvider.tsx
+│       ThresholdsContextProvider.tsx
+│       UnitsOfMeasureContextProvider.tsx
+│       UserContext.tsx
+│       YearlyReportContextProvider.tsx
+│
+├───interfaces
+│       addressbook_interfaces.tsx
+│       alert_interface.tsx
+│       capacitygroup_interfaces.tsx
+│       company_interfaces.tsx
+│       customer_interfaces.tsx
+│       customoption_interface.tsx
+│       demand_interfaces.tsx
+│       event_interfaces.tsx
+│       favorite_interfaces.tsx
+│       infomenu_interfaces.tsx
+│       supplier_interfaces.tsx
+│       threshold_interfaces.tsx
+│       user_interfaces.tsx
+│
+└───util
+        Api.tsx
+        Auth.tsx
+        AuthApi.tsx
+        AuthenticatedRoute.tsx
+        Defaults.tsx
+        ErrorMessagesHandler.tsx
+        RefreshToken.tsx
+        TypeGuards.tsx
+        WeeksUtils.tsx
+```
+
+
+
+## NOTICE
+
+This work is licensed under the [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+- SPDX-License-Identifier: Apache-2.0
+- Licence Path: https://creativecommons.org/licenses/by/4.0/legalcode
```

Comment:
This is not the Apache-2.0 License reference. 
Is it Apache-2.0 or CC-BY-4.0?
See [TRG 7.07](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07#documentation) for guidance on proper Notices

Reply:
Fixed via commit:
https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/140ce7933a091d2624344ae74cb4a936c32e9419